scripts/zones/Xarcabard/Zone.lua

```diff
@@ -5,6 +5,7 @@ local ID = require('scripts/zones/Xarcabard/IDs')
 require('scripts/quests/i_can_hear_a_rainbow')
 require('scripts/globals/conquest')
 require('scripts/globals/keyitems')
+require("scripts/globals/teleports")
```

Comment:
Single quotes in these includes, like all the others its surrounded by

Reply:
fixed =)